modules/rapidyaml/0.9.0/patches/build.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/BUILD b/BUILD
+new file mode 100644
+index 0000000..65dd58a
+--- /dev/null
++++ b/BUILD
+@@ -0,0 +1,11 @@
++package(default_visibility = ["//visibility:public"])
++
++cc_library(
++    name = "ryml",
```

Comment:
nit: You can name the target `rapidyaml` so that this target can be used by other modules just by `@rapidyaml` instead of `@rapidyaml//:ryml`.

Reply:
Thanks, fixed.